src/service_control_client_impl_test.cc

```diff
@@ -1892,6 +1897,7 @@ TEST_F(ServiceControlClientImplTest,
   ServiceControlClientOptions options(
       CheckAggregationOptions(1 /*entries */, 500 /* refresh_interval_ms */,
                               1000 /* expiration_ms */),
+      QuotaAggregationOptions(1 /*entries */, 500 /* refresh_interval_ms */),
       ReportAggregationOptions(1 /* entries */, 500 /*flush_interval_ms*/));
```

Comment:
Your added Quota() function at client.h level should be unit-tested here.
This test file is too big now.  you can add a new one service_control_client_impl_quota_test.cc
and split the current one into two:  xxxx_check_test.cc  and xxx_report_test.cc

You can do this in a new PR.